.github/workflows/publish.yml

```diff
@@ -25,7 +28,13 @@ jobs:
         run: npm install
       - name: Build
         run: npm run build --workspaces
+      - name: Bump patch version
```

Comment:
@upendranadh-cruxito Auto patch logic in the works here - which means we no longer need to manually version our patches.

Reply:
@perceeg This will be helpful Thank you !